### DISCUSSION TITLE
Data calculation lasts for days

### DISCUSSION THREAD

**Discussion by DAGorchakov** [minio/minio#15049] [Q&A] [answered]
## NOTE
I have uploaded data with Web console to check if alert would work (`avg by(bucket) ((minio_bucket_usage_total_bytes * 100) / minio_bucket_quota_total_bytes) > 90`).  Limit on bucket is 9.5MiB, I have uploaded 114MiB yesterday. minio metrics in Prometheus have been showing that my bucket is empty still.   
How long it takes to recalculate data?   
Can I force data recalculation in a certain bucket?  
![image](https://user-images.githubusercontent.com/17656075/172337763-5d140cee-571d-408a-883c-dda8cb629555.png)

## Your Environment

* Version used (`minio --version`): `2022-05-26T05:48:41Z`
* Server setup and configuration: "Red Hat Enterprise Linux Server 7.9 (Maipo)"
* Operating System and version (`uname -a`): Linux name 3.10.0-1160.59.1.el7.x86_64 1 SMP Wed Feb 16 12:17:35 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Comment by klauspost** [comment, marked as answer]
Recalculation depends on your system speed and the total number of objects. Last count was 8M as per your screenshot.

If you want you can increase the scanner speed with `mc admin config set ALIAS scanner delay=0` at the cost of more resources. Default is '10'. Higher is slower.

**Reply by DAGorchakov** [reply]
Thank you for reply!
How do you know that `Last count was 8M as per your screenshot`, pls show me, where it is?
Maybe there are some kind of docs for this issue, can you share the link to it? I would learn it

**Reply by DAGorchakov** [reply]
I see https://docs.min.io/docs/minio-server-configuration-guide.html

**Reply by klauspost** [reply]
![image](https://user-images.githubusercontent.com/5663952/172348386-a8c89b1a-5f20-426e-ab80-11d4bbc5dbf3.png)
